plugins/pluginmanagers/lazy.nix

```diff
@@ -162,7 +163,7 @@ in
   config = mkIf cfg.enable {
     extraPlugins = [ cfg.package ];
 
-    extraPackages = [ cfg.gitPackage ];
+    dependencies.git.enable = lib.mkDefault true;
```

Comment:
This ends up being undocumented at a per-plugin level. Before it was documented by the `plugins.*.fooPackage` option itself.

For now, we probably need a note in the plugin's description (or enable option?) mentioning that enabling this plugin will enable `dependencies.foo.enable` by default.